scripts/generate_release_notes.py

```
@@ -0,0 +1,330 @@
+Generate professional, user-focused release notes now:
+"""
+    
+    try:
+        client = genai.Client(api_key=api_key)
+        
+        response = client.models.generate_content(
+            model="gemini-2.0-flash-exp",
+            contents=prompt,
+        )
+        
+        return response.text
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)